charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.75.0
+version: 0.76.0
```

Comment:
please bump the patch version, not the minor version. Once you've done that, please run `make generate-examples CHARTS=opentelemetry-operator`

Reply:
@jtyr it looks like there's no patch version bump anymore, can you do that and re-run the make generate when you can please?